src/chart/generateCategoricalChart.tsx

```diff
@@ -1027,6 +1032,15 @@ export const generateCategoricalChart = ({
       if (!_.isNil(this.props.syncId)) {
         this.addListener();
       }
+
+      this.accessibilityManager.container = this.container;
+      this.accessibilityManager.offset = {
+        left: this.props.margin.left ?? 0,
+        top: this.props.margin.top ?? 0,
+      };
+      this.accessibilityManager.coordinateList = this.state.tooltipTicks;
+      this.accessibilityManager.mouseHandlerCallback = this.handleMouseMove;
+      this.accessibilityManager.layout = this.props.layout;
```

Comment:
can we init the `AccessibilityManager` here and set all of this in a constructor instead of one at a time?